### ISSUE TITLE
Create Order

### ISSUE THREAD

**Issue by jkillz2020** [Team-Backrow/Terminal-Interface-New#5]
Add Objects from ProductsTable to Order.  THIS IS A JOIN TABLE!
obj.OrderId [key],
obj.CustomerId

**Comment by devoisaurus** [comment]
Won't this need the product id too?

**Comment by jkillz2020** [comment]
yes.  I was just typing that on the fly, not an accurate example.

**Comment by devoisaurus** [comment]
obj.OrderId [key],
obj.CustomerId,
obj.PaymentId,
obj.TotalPrice

**Comment by devoisaurus** [comment]
Submitted PR with properties setup for Order class

**Comment by blairvansant** [comment]
👍
